### DISCUSSION TITLE
Rebasing errors => "We're looking into it"

### DISCUSSION THREAD

**Discussion by brcarp** [community/community#43773] [Code Security]
### Select Topic Area

General

### Body

I've been trying to do `@dependabot rebase` for about a day, and with each failure I get a message that says, "Dependabot tried to update this pull request, but something went wrong. We're looking into it, but in the meantime you can retry the update by commenting @dependabot rebase."

Because of the "we're looking into it", should I feel confident that the Dependabot team is aware of the failures and able to act on it? There's not a lot of feedback beyond that, and as far as I know there's no dedicated "Dependabot status" page nor a section for it on [githubstatus.com](https://www.githubstatus.com/).

**Comment by Mutebifred** [comment]
Thank you my friend!
